docs/schema.md

```diff
@@ -34,9 +34,8 @@ Top-level: `object`
               - Version: `string` (semver version specifier)
         - Artifacts: `object[]`
           - URL: `string` (download url of the .dll file)
-          - Filename: `string` (filename to use for the file)
           - Sha256: `string`
-          - Install location: `string` (defaults to "/nml_mods")
+          - Install location: `string` (defaults to "/nml_mods/", if doesn't end in slash use last part as a custom filename)
```

Comment:
This I'm still unsure about. The slash thing seems like it could be easily missed in PR reviews.

Reply:
That is true, but it seems quite jarring to have two separate fields for dealing with essentially the same thing, the file's path.

I guess that there's an expectation that the filename will be by default taken from the download URL after the last `/` character for the vast majority of cases?

In which case I can agree with it being bad to make the common case worse than the rare one. But then it should also be documented how the filename should be gotten in cases where it's not manually defined, as `Content-Dispotision` is a standard HTTP header which I imagine would need to have it's filename directive ignored for the file paths to be auditable from the download URLs.

Reply:
I feel like the whole .dll matching doesn't make sense, as there's many cases where you might for example want to have other types, such as json, yaml, toml, png, etc. 

The filename fs thing... I feel like whenever merging PRs to accept manifests the filenames should just be limited to reasonably ASCII ones without a good reason to not to do so (so `[A-Za-z0-9\-\_]+\.[A-Za-z0-9]+` type of a thing)?

Reply:
Oh yeah, good point. The dll matching is an awful idea.